content/en/dora_metrics/setup/deployments.md

```diff
@@ -116,6 +116,18 @@ There are two requirements for calculating Change Lead Time:
 1. Both the Git repository URL and commit SHA are provided when sending deployment events.
 2. Your repository metadata is being [synchronized to Datadog](#synchronize-repository-metadata-to-datadog).
 
+### Breakdown metrics
+
+DataDog also provides the following breakdown metrics, which represent the different stages since the commit is made until it is deployed.
+
+- `dora.time_to_pr_ready`: Time from the commit until the PR is ready for review. This metric is only emitted for commits that were made before the PR is ready for review.
```

Comment:
The first 3 metrics are also only available if a commit is related to a PR right?

Reply:
Yes, I have added a comment with this.